### DISCUSSION TITLE
mcu 'mcu': Unknown command: t5uid1_ping

### DISCUSSION THREAD

**Discussion by napter** [Desuuuu/DGUS-reloaded-Klipper#28] [Q&A]
I am trying to connect a BTT SKR Mini E3 V2 board to the stock display. I am using desuuuu klipper and built the firmware with the following settings:
![image](https://user-images.githubusercontent.com/6260841/160942473-35ee706a-de9e-472c-8da4-9fed655e8e7b.png)

When I connect I get this error, for which I cannot find any information:
  mcu 'mcu': Unknown command: t5uid1_ping
  
  This type of error is frequently caused by running an older
  version of the firmware on the micro-controller (fix by
  recompiling and flashing the firmware).
  
  Known versions: host=dgus-reloaded-211225-0-g4e2090f8, mcu=?-20220320_061608-dd11e99d10e8
  
  Once the underlying issue is corrected, use the "RESTART"
  command to reload the config and restart the host software.
  Protocol error connecting to printer

I tried changing the screen serial port to USART1 and USART3 to see if that helped. Got the same error with USART3 and USART1 gave a different error.

Any help would be appreciated

**Comment by Cortii2** [comment]
It was the same for me yesterday! Unknown command: t5uid1_ping

I commented out [t5uid1] part of the config, the board sent back ready status, BUT! The screen is stucked to the boot logo :o 
Sooo, on both side, on the board and on the screen I also have Desuuuu version FWs and screen is not working :(((
